lib/admin/WP_Auth0_Admin_Features.php

```diff
@@ -423,19 +423,24 @@ public function security_validation( $old_options, $input ) {
 					__( 'No database connections found for this application. ', 'wp-auth0' ) .
```

Comment:
Can't leave a comment for the line 416 
```
 		if ( $old_options['password_policy'] !== $input['password_policy'] ) {
```
but seems like you could check the opposite value and return early instead of nesting everything here.

Reply:
I could but minimal changes for the patch ... I don't want to go crazy refactoring.